packages/amplify-e2e-tests/src/schema-api-directives/tests/predictions-usage.ts

```diff
@@ -34,7 +34,7 @@ export async function runTest(projectDir: string, testModule: any) {
     expect(result).toBeDefined();
     const pollyURL = result.data.speakTranslatedImageText;
     // check that return format is a url
-    expect(pollyURL).toMatch(/(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/);
+    expect(pollyURL).toMatch(/(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-/]))?/);
```

Comment:
I'm not going to try to pick apart this regex right now, but this seems like a functional change?

Reply:
A backslash in that location is just escaping the forward slash after it, which does not need escaping.